src/redux/settings.ts

```diff
@@ -106,6 +121,33 @@ export const settingsLoadNetwork = () => async (
   }
 };
 
+export const settingsLoadTestnetsEnabled = () => async (
```

Comment:
this function isn't called anywhere - you can just do the dispatch as part of `settingsLoadState` - you may want to create a new action for "load success" that updates both the native currency and the testnets pref, and just delete this function.